### ISSUE TITLE
Add support for tool tip text using the title attribute

### ISSUE THREAD

**Issue by brantwills** [brantwills/Angular-Paging#36]
As per issue #31:
It will be good to have something like a tooltip-func attribute to which callback function can be passed.

**Comment by brantwills** [comment]
Included a new directive attribute `text-title-page`
This allows the user to create custom hover over title text and reference the page 
Still need to update the readme files for instructions on how to use 
I will keep this open until all the docs have been updated


Sample Usage: 
```html
<paging text-title-page="Anytime you type {page} you get the page number!"></paging>
```

**Comment by brantwills** [comment]
Docs have been updated
Closing with v2.1.0 release

**Comment by johnknoop** [comment]
Can't get this feature to work. It just says "Page {n}" reglardless of what I put in the text-title-page attribute.

**Comment by brantwills** [comment]
Hey johnknoop sorry to hear it is not working as expected.  
I have not encountered this `{n}` issue yet, could you provide a description of how you are using the directive?

**Comment by johnknoop** [comment]
Hi @brantwills 

I didn't meen literally {n}. It says "Page" and then whatever page number it might be. But I want a completely different text (i.e. in a different language).

**Comment by brantwills** [comment]
Ahhh I see!
Mmm, sounds like the `default` settings are overriding your values :(

I've created a plunker to explore the issue:
http://plnkr.co/edit/nqBwA5zqdTzMq5Y8wA7Z?p=preview

I was able to successfully input new text in the `text-title-page` attribute 
However, feel free to update the code to better suite your environment

I'm thinking there may be a unique condition you have which is trigging the override?
Perhaps your paging directive is contained in an `ng-if` or `ng-repeat` or something?
